cczoo/rag/README.md

```diff
@@ -0,0 +1,97 @@
+# Retrieval Augment Generation with Intel&#174; TDX
+This repo contains code to build various chatbot services using the state-of-the-art large language models.
+
+## Getting Started 
+
+### 1. Prepare for Docker images
+You can download the Docker images from `docker.io`:
+
+```bash
+docker pull intelcczoo/tdx-dev:dcap_mvp2023ww15-ubuntu22.04-latest
+docker pull intelcczoo/rag-llm:release
+docker pull intelcczoo/rag-llm:ui
+```
+
+Or you can compile Docker images locally:
+
+**Notice: Please contact siyuan.hui@intel.com to obtain the dcap package required to build Docker image.**
```

Comment:
it will be better to use the released DCAP packages in 01.org, not from personal.